Add instructions

anssiko opened the pull request:
@dontcallmedom @cvan let's use this PR to hash out the instructions, see what should be included. Perhaps this should be on a separate page, but let's get the content ready first.

Preview: https://rawgit.com/w3c/vr-workshop/instructions/schedule.html

cvan committed:
fix + enhance anchor links; make nav + layout more responsive; restructure few elements

dontcallmedom committed:
Update to participants list

anssiko committed:
Fix schedule page anchor navigation

anssiko committed:
Fix participants page navigation

dontcallmedom committed:
Remove JeffS from participants list

:(

dontcallmedom committed:
Update participant list

dontcallmedom committed:
Fix name in participants list

anssiko committed:
Add instructions to schedule page

cvan commented on schedule.html:
```diff
@@ -315,6 +355,111 @@ <h4 id="schedule-day-2">
+          <span class="lightning" id="lightning"></span>
+        </dt>
+        <dd>
+          Discussion topic (30-60 mins) starts off with 3-4 5-minute <a href=
```
would change to `three or four 5-minute`

cvan replied:
`mins` and `minute` - should be consistent

dontcallmedom replied:
addressed

cvan commented on schedule.html:
```diff
@@ -315,6 +355,111 @@ <h4 id="schedule-day-2">
+        <dd>
+          Participants break into smaller breakout groups to discuss more
+          specific topics over a period of maximum 60 minutes. The breakout
+          session schedule grid is identified and built collaboratively by the
```
would remove word `grid`

dontcallmedom replied:
addressed

cvan commented on schedule.html:
```diff
@@ -315,6 +355,111 @@ <h4 id="schedule-day-2">
+          Participants break into smaller breakout groups to discuss more
+          specific topics over a period of maximum 60 minutes. The breakout
+          session schedule grid is identified and built collaboratively by the
+          participants in the course of the workshop. Each groups to provide a
```
`groups` to `group is`

cvan commented on schedule.html:
```diff
@@ -315,6 +355,111 @@ <h4 id="schedule-day-2">
+      </ul>
+      <p>
+        The <a href="https://w3c-vr.slack.com/messages/general/">#general
+        channel</a> is used for plenary and <a href="#lightning">lightning
```
link to `#plenary` as well?

dontcallmedom replied:
addressed

cvan commented on schedule.html:
```diff
@@ -315,6 +355,111 @@ <h4 id="schedule-day-2">
+      </h4>
+      <p>
+        Meeting notes will be recorded from all the sessions summarizing the
+        discussions. These notes will be made public. TODO: add instructions
```
perhaps make `TODO` a comment instead?

cvan commented on style.css:
```diff
@@ -366,10 +366,62 @@ table.schedule {
   font-size: 67%;
 }
 
+span.lightning, span.plenary, span.breakout, span.panel {
```
might as well create a common class for these

cvan commented on style.css:
```diff
@@ -366,10 +366,62 @@ table.schedule {
+}
+
+span.plenary:before {
+  content: "plenary"
```
`;`

cvan commented on schedule.html:
```diff
@@ -315,6 +355,111 @@ <h4 id="schedule-day-2">
+        <dd>
+          Discussion topic (30-60 mins) starts off with 3-4 5-minute <a href=
+          "https://en.wikipedia.org/wiki/Lightning_talk">lightning talks</a> by
+          speakers nominated in advance. Lightning talks are followed by group
```
this is mentioned already [above](https://github.com/w3c/vr-workshop/pull/40/files#diff-0953deb6637fd0edbb59e4e408606954R371), but I suppose it's fine to repeat it so people don't miss

cvan commented on schedule.html:
```diff
@@ -315,6 +355,111 @@ <h4 id="schedule-day-2">
+          <a href="papers.html">Position statements</a>
+        </li>
+        <li>
+          <a href="https://surma.link/things/reading-specs/">How to read web
```
good link to add 👍

cvan reviewed (changes requested):
👍 looks great. just some minor changes.

I especially like what you did with the colour-coding and legend. easy to read.

cvan commented on schedule.html:
```diff
@@ -315,6 +355,111 @@ <h4 id="schedule-day-2">
+      </h3>
+      <dl class="format">
+        <dt>
+          <span class="plenary" id="plenary"></span>
```
seems like it'd be better to just do

``` html
<dt class="format format-plenary" id="plenary">
  Plenary
</dt>
```

cvan committed:
fix spacing in few participant names

cvan commented on style.css:
```diff
@@ -366,10 +366,62 @@ table.schedule {
+span.lightning, span.plenary, span.breakout, span.panel {
+  display: block;
+  margin: 0 5px 0 0;
+  font-size: 67%;
```
this looks good in the schedule, but for the legend, the font-size could be bumped up for legibility and some margins could be added I think.

<img width="1153" alt="screen shot 2016-10-11 at 1 12 33 pm" src="https://cloud.githubusercontent.com/assets/203725/19286901/67601718-8fb4-11e6-8bdb-55fe8aedfda3.png">

<img width="1154" alt="screen shot 2016-10-11 at 1 10 31 pm" src="https://cloud.githubusercontent.com/assets/203725/19286886/558eab62-8fb4-11e6-8c0f-c4da2ced918b.png">

here are the styles I used:

``` css
dt.format {
  box-shadow: .3em .3em 0 rgba(0,0,0,.1);
  font-size: 1rem;
  font-weight: 600;
  margin: 0 5px 15px 0;
  padding: 8px 10px;
}
```

dontcallmedom replied:
addressed

cvan commented on style.css:
```diff
@@ -366,10 +366,62 @@ table.schedule {
+}
+
+span.lightning:before {
+  content: "lightning";
```
IMO, we should just add normal text content

cvan replied:
(mostly for copy+paste reasons and accessibility)

dontcallmedom replied:
addressed

dontcallmedom committed:
Update participant lists with people from waiting list

dontcallmedom committed:
Update participant list

dontcallmedom committed:
Add another remote participant

dontcallmedom committed:
Improve accessibility of schedule session types

dontcallmedom committed:
Tweaks to the description of the schedule

as suggested by @cvan

anssiko committed:
Merge Authoring VR & Declarative VR sessions

anssiko committed:
Add keynote speaker

anssiko committed:
Tentative Schedule -> Schedule

anssiko committed:
Tidy

anssiko committed:
Complete meeting notes instructions

dontcallmedom committed:
Use collapsable items for schedule

dontcallmedom commented:
I've changed the markup for the schedule to allow to add more detailed informations (speakers, related papers, links to slides) in collapsible form

anssiko commented:
@dontcallmedom @cvan thanks for your review comments and commits!

I think we're now ready to go live with these "instructions v1" (can add more instructions later as needed).

@dontcallmedom WDYT?

dontcallmedom commented:
@anssiko I'll merge the pre-markup changes to gh-pages and keep that PR open for further iterations

anssiko commented:
@dontcallmedom Stop press. I think https://github.com/w3c/vr-workshop/pull/40/commits/6fc2dd30b79c314aa9da6b5144cbdd5a3f7dc24d broke the CSS?

anssiko commented:
See https://rawgit.com/w3c/vr-workshop/instructions/schedule.html

dontcallmedom commented:
can you be more specific on "broke the CSS" @anssiko ?
(note that as described above, I've merged the clean version to gh-pages)

anssiko commented:
OK, now I see you did not merge the experimental collapsable items feature yet -- good.

Let's keep iterating that in this PR and integrate the details as well.

dontcallmedom committed:
Add missing link on opening remarks

dontcallmedom committed:
Reinstate break specific style

dontcallmedom committed:
Ensure alignment in schedule

dontcallmedom committed:
Add some of the details to some of the sessions

dontcallmedom commented:
oh, I guess the rendering is very different in chrome and in firefox; I now understand better the broken style comment :)

dontcallmedom committed:
Fix markup

dontcallmedom commented:
ok, apparently, [`summary` cannot be used as a target for `display:flex` in Chrome](http://stackoverflow.com/questions/30650471/display-flex-doesnt-work-on-summary-tags-in-chrome), will fix

dontcallmedom committed:
Fix interop of collapsible schedule

Per http://stackoverflow.com/questions/30650471/display-flex-doesnt-work-on-summary-tags-in-chrome and http://www.otsukare.info/2016/04/19/summary-details

dontcallmedom committed:
Make detailed view more usable for small screens

anssiko commented on schedule.html:
```diff
@@ -92,229 +90,193 @@ <h3 id="silver-sidebar">
-        <b>This schedule may change based on discussions with the <a href=
-        "./#program">program committee</a>.</b>
+        We use Slack during the workshop (offers a lot of IRC-like features).
+        To join the workshop Slash team:
```
s/Slash/Slack/

cvan replied:
fixed

dontcallmedom committed:
Tweak look of detailed view

dontcallmedom committed:
Document another scribe

dontcallmedom committed:
Tweak some more the bubble style

anssiko committed:
Make Authoring VR one slot, extend Perf, add Demo time

anssiko commented:
Cherry-picking https://github.com/w3c/vr-workshop/commit/defa3619b97a5d077d810fa95c5b1c114852a710 was too painful, so we have a conflict to resolve.

cvan commented:
do you want me to manually resolve + merge this?

anssiko commented:
@cvan if you can resolve the conflict introduced by https://github.com/w3c/vr-workshop/commit/defa3619b97a5d077d810fa95c5b1c114852a710 that'd be awesome. I wouldn't yet merge this branch into gh-pages since there are bugs, and @dontcallmedom indicated he's planning to fix them.

cvan committed:
Merge branch 'gh-pages' into instructions

anssiko commented:
To accommodate Philip Rosedale I switched two slots. @cvan It'd be great if you could integrate https://github.com/w3c/vr-workshop/commit/ca7a642c07794db0e2b6c8b50f953a7d7226e65f to this branch.

cvan committed:
fix various issues with Schedule styles; fix for mobile

cvan commented:
closing in favour of new upcoming PR
